src/__tests__/__snapshots__/config.test.js.snap

```diff
@@ -177,6 +177,7 @@ Object {
     },
   ],
   "jsDelivrHitsEndpoint": "https://data.jsdelivr.com/v1/stats/packages/npm/month/all",
+  "jsDelivrPackageEndpoint": "https://data.jsdelivr.com/v1/package/npm",
```

Comment:
let's remove this snapshot and just assert nothing changed in the test, instead of needing to update this every time. It seems fairly useless

Reply:
(in a separate PR)

Reply:
yes 👌